interpreter.py

```diff
@@ -14,14 +15,12 @@ def jitpolicy(driver):
     from rpython.jit.codewriter.policy import JitPolicy
     return JitPolicy()
 
-jit_driver = jit.JitDriver(greens=[], reds=[])
+jit_driver = jit.JitDriver(greens=["self", "block", "function"],
```

Comment:
@cfbolz what do you reckon to having `function` in the greens? Presumably a block cannot be shared across more than one function anyway...